src/server/pfs/cmds/cmds.go

```diff
@@ -789,6 +790,9 @@ $ {{alias}} repo@branch -i file
 # files into your Pachyderm cluster.
 $ {{alias}} repo@branch -i http://host/path`,
 		Run: cmdutil.RunFixedArgs(1, func(args []string) (retErr error) {
+			if noProgress {
+				progress.Disable()
```

Comment:
As with #5654, you could use https://github.com/mattn/go-isatty to disable the progress bars when writing to a non TTY output as well